src/main/java/com/code_intelligence/jazzer/driver/FuzzTargetRunner.java

```diff
@@ -368,6 +368,15 @@ public static int startLibFuzzer(List<String> args) {
       // https://github.com/llvm/llvm-project/blob/da3623de2411dd931913eb510e94fe846c929c24/compiler-rt/lib/fuzzer/FuzzerFlags.def#L19
       args.add("-len_control=100");
     }
+
+    for (String arg : args.subList(1, args.size())) {
+      if (!arg.startsWith("-")) {
+        String message = "using inputs from: ";
+        message += arg;
+        Log.info(message);
```

Comment:
```suggestion
        Log.info("using inputs from: " + arg);
```

Reply:
@zgtm Small cleanup, PTAL

Reply:
Ah, yes, thanks for the hint … I have written too much rust it seems! ;)